test_files/namespaced/reopen_ns.ts

```diff
@@ -0,0 +1,12 @@
+export namespace ns {
+  export let x = 0;
+}
+
+export namespace ns {
+  export let y = 0;
+}
+
+// this implicitly re-emits exports.ns, so it is in a way reopening ns.
```

Comment:
Sorry, I don't understand how this implicitly re-emits the other namespace.

Reply:
see now